lib/v2/screens/explore_screens/invite/interactor/viewmodels/bloc.dart

```diff
@@ -0,0 +1,3 @@
+export 'invite_bloc.dart';
```

Comment:
we can use `part of` and just export the bloc.

Reply:
Any benefit over the current barrel file?